jwst/outlier_detection/outlier_detection_step.py

```diff
@@ -26,8 +26,9 @@
 class OutlierDetectionStep(Step):
     """Flag outlier bad pixels and cosmic rays in DQ array of each input image.
 
-    Input images can be listed in an input association file or already opened
-    with a ModelContainer.  DQ arrays are modified in place.
+    Input images can be listed in an input association file or dictionary,
+    or already opened with a ModelContainer or ModelLibrary.
+    DQ arrays are modified in place.
```

Comment:
SCI, ERR, and VAR arrays are also updated here, with NaN values matching the DQ flags.